client/src/components/app.jsx

```diff
@@ -88,12 +107,19 @@ class App extends React.Component {
             clickFunction={this.previousSlide}
             glyph="&#9664;" />
 
+          <div className="exit-button" onClick={() => this.handleExitClick()}><span>X</span></div>
```

Comment:
No need for anonymous function wrapper when function is already bound in the constructor.  DRY!